src/itinerary/Itinerary.tsx

```diff
@@ -110,7 +120,7 @@ const Itinerary = ({
                     : place.mainLabel
                 }
               />
-              <Component {...hrefProps}>
+              <Component {...hrefProps} aria-label={place.actionLabel}>
```

Comment:
What happens if place.stepLabel is undefined?
Also, actionLabel should probably be in our Place definition (https://github.com/blablacar/ui-library/blob/master/src/typings/place.d.ts)

Reply:
As a "spectator" it's weird that this conversation is marked as "resolved" when there was no reply...

I guess the answer is that the attribute is not rendered if it's undefined, but still.